ui/src/app/store/sshkey/reducers.test.ts

```diff
@@ -0,0 +1,254 @@
+import {
+  sshKey as sshKeyFactory,
+  sshKeyState as sshKeyStateFactory,
+} from "testing/factories";
+import reducers, { actions } from "./slice";
+
+describe("sshkey reducer", () => {
+  it("should return the initial state", () => {
+    expect(reducers(undefined, { type: "" })).toEqual(sshKeyStateFactory());
+  });
+
+  it("should correctly reduce fetchStart", () => {
```

Comment:
Have been using the form `it("reduces fetchStart")` for the new reducer tests, a bit more succinct, but hardly important.